pkg/apis/policy/v1alpha1/propagation_types.go

```diff
@@ -173,6 +173,12 @@ type PropagationSpec struct {
 	// +kubebuilder:validation:Enum=Lazy
 	// +optional
 	ActivationPreference ActivationPreference `json:"activationPreference,omitempty"`
+
+	// OrphaningDeletion  represents whether to prevent removal of these managed resources
+	// +kubebuilder:default="Never"
+	// +kubebuilder:validation:Enum=Never;Complete;Incomplete
+	// +optional
+	OrphaningDeletion OrphaningDeletion `json:"orphaningDeletion,omitempty"`
```

Comment:
The first thing that strikes users about this name is that its value is a bool. Is there a better name for it?

Reply:
> The first thing that strikes users about this name is that its value is a bool. Is there a better name for it?

how about  OrphaningDeletionPolicy

Reply:
I think it's okay. Let me give one for reference: `PropogateDeletionPolicy`.

/cc @chaunceyjiang @zhy76 @RainbowMango

Reply:
> how about OrphaningDeletionPolicy

+1